go/oasis-test-runner/oasis/compute.go

```diff
@@ -99,7 +104,9 @@ func (worker *Compute) startNode() error {
 	var runtimeArray []*Runtime
 	if len(worker.runtimes) > 0 {
 		for _, idx := range worker.runtimes {
-			runtimeArray = append(runtimeArray, worker.net.runtimes[idx])
+			if idx >= 0 {
```

Comment:
What's the reason for having negative indices instead of just omitting those indices?

Reply:
Since current logic is that if no runtimes are set, it will use all runtimes configured in the network cfg: https://github.com/oasisprotocol/oasis-core/blob/02a2481df9a8a93d3faa1df8a7f65978dd763bb9/go/oasis-test-runner/oasis/compute.go#L100-L106

Can look into how many test cases rely on that, and fix it up across scenarios, to not need the negative index hack.

Reply:
fixed